<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="viewLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>851</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>添加你的图片！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolBox" name="toolPage">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>457</height>
      </rect>
     </property>
     <attribute name="label">
      <string>手动标注</string>
     </attribute>
     <widget class="QPushButton" name="changeToManuallyLable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>切换至标注模式</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>457</height>
      </rect>
     </property>
     <attribute name="label">
      <string>自动标注</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>导入模型</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>更换模型</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>自动标注（单击我标注）</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>457</height>
      </rect>
     </property>
     <attribute name="label">
      <string>已有标签</string>
     </attribute>
     <widget class="QTextBrowser" name="currentLabelCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>151</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="curPlaceProcessBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="processTextLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;当前你在 / &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoAfterOne">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoPrevious">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeCurPictureBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>移除当前的图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeCurPicBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>640</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>更换当前图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageLabelButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>570</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>调教你的标签</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_newPicture"/>
    <addaction name="action_newDirectory"/>
    <addaction name="actionswitchCurPicture"/>
    <addaction name="actionswitchCurDir"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作图片...</string>
    </property>
    <addaction name="actionchangeLightness"/>
    <addaction name="actionchangeContrary"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>删除图片...</string>
    </property>
    <addaction name="actionremoveSinglePicture"/>
    <addaction name="actionremoveAllPictures"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>管理标签</string>
    </property>
    <addaction name="actionimportlabelIndex"/>
    <addaction name="actionaddNewLabel"/>
    <addaction name="actiondeleteOldLabel"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="checkQuickKeySequence"/>
    <addaction name="actionfindShitBugAndsakForMail"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>关于...</string>
    </property>
    <addaction name="actioncheckAuthor"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_newPicture">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="action_newDirectory">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionchangeLightness">
   <property name="text">
    <string>亮度调整</string>
   </property>
  </action>
  <action name="actionchangeContrary">
   <property name="text">
    <string>调整对比度</string>
   </property>
  </action>
  <action name="actionsaveAsPng">
   <property name="text">
    <string>导出为png</string>
   </property>
  </action>
  <action name="actionsaveAsJpg">
   <property name="text">
    <string>导出为jpg</string>
   </property>
  </action>
  <action name="actiongetLablePositionAsTxt">
   <property name="text">
    <string>生成txt标注数据</string>
   </property>
  </action>
  <action name="actiongetLablePositionAsYaml">
   <property name="text">
    <string>生成yaml格式的数据</string>
   </property>
  </action>
  <action name="actionremoveSinglePicture">
   <property name="text">
    <string>移除当前图片</string>
   </property>
  </action>
  <action name="actionremoveAllPictures">
   <property name="text">
    <string>移除所有图片</string>
   </property>
  </action>
  <action name="actionswitchCurPicture">
   <property name="text">
    <string>切换图片</string>
   </property>
  </action>
  <action name="actionswitchCurDir">
   <property name="text">
    <string>切换文件目录</string>
   </property>
  </action>
  <action name="actionimportlabelIndex">
   <property name="text">
    <string>导入已有标签</string>
   </property>
  </action>
  <action name="actionaddNewLabel">
   <property name="text">
    <string>添加新标签</string>
   </property>
  </action>
  <action name="actiondeleteOldLabel">
   <property name="text">
    <string>删除旧标签</string>
   </property>
  </action>
  <action name="checkQuickKeySequence">
   <property name="text">
    <string>查阅快捷键</string>
   </property>
  </action>
  <action name="actioncheckAuthor">
   <property name="text">
    <string>看看这玩意谁做的</string>
   </property>
  </action>
  <action name="actionfindShitBugAndsakForMail">
   <property name="text">
    <string>发现了bug然后问候作者</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
